docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -1,4 +1,4 @@
-= Creating secrets for your builds
+= Creating secrets for your pipelines
```

Comment:
I don't disagree with the accuracy of this change, but I think it muddies the intention. This is in the section "Configuring your builds" along with all the rest of the build pipeline configurations. Should we move this section out?

I think that these secrets are called build-time secrets in the UI, but that was to differentiate between the secrets that we used to have to deployments.

Reply:
Resolved in 4a296cc.